projects/templates/projects/done_test.html

```diff
@@ -16,7 +16,39 @@
 {% block content %}
     <div >
         <p>Project</p>
+        <li>{{ project_data.id }}</li>
+        <li>{{ project_data.name }}</li>
+        <li>{{ project_data.description }}</li>
+        <li>{{ project_data.due_date }}</li>
+        {% if project_data.status == 0 %}
+            <li>NOT STARTED</li>
+        {% elif project_data.status == 10 %}
+            <li>IN PROGRESS</li>
+        {% elif project_data.status == 20 %}
+            <li>UNDER REVIEW</li>
+        {% elif project_data.status == 30 %}
+            <li>COMPLETED</li>
+        {% else %}
+            <li>STALLED</li>
+        {% endif %}
+        </br>
         <p>is Done!</p>
+        <br></br>
+        <p>Features</p>
+        {% for feature in feature_data %}
+            {% if feature.project.id == project_data.id %}
```

Comment:
Instead of looping through all Features and checking if `feature.project.id` matches, just loop through the list of ManyToOne elements directly.

https://docs.djangoproject.com/en/3.0/topics/db/examples/many_to_one/